### PR TITLE
Add lcms port

### PR DESCRIPTION
**Pull request by donny-dont**
Little-CMS is used in WebKit and also image libraries.

### CODE REVIEW

**Comment by unicodon** [comment]
I needed to install pkgconfig stuff so that libjxl could find lcms2. 
https://github.com/unicodon/WebKitRequirements/commit/18d5586c4180b0ccafd11a24b8cfa6e28c9ff194

**Comment by donny-dont** [comment]
My plan was to swipe the FindLCMS2 from WebKit and add it to the jxl build.

**Comment by unicodon** [comment]
I'll try this out with my libjxl branch.

**Comment by unicodon** [comment]
Instead of patching we may want to have build/CMakeLists.txt and copy it, like https://github.com/WebKitForWindows/WebKitRequirements/blob/main/ports/pixman/portfile.cmake#L25.

File: ports/lcms/patches/0001-Add-CMake-build.patch [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
++    VERSION 2.12.0
++    LANGUAGES C
++)
++
++include(CheckSymbolExists)
++include(GNUInstallDirs)
++
++add_library(lcms2 "")
++target_sources(lcms2 PRIVATE
++    src/cmsalpha.c
```
Comment (unicodon) [inline ports/lcms/patches/0001-Add-CMake-build.patch:29]: It'd be nice to get this list sorted.

Comment (donny-dont) [reply ports/lcms/patches/0001-Add-CMake-build.patch:29]: I was just using the listing in the Meson port but I'll go ahead and sort it all.

**Review by unicodon — APPROVED** [review]

**Pull request merged**